identity/src/main/java/com/android/identity/Constants.java

```diff
@@ -146,4 +173,26 @@ public class Constants {
     public @interface BleDataRetrievalOption {
     }
 
+    /**
+     * An annotation used to specify allowed curve identifiers.
+     *
+     * <p>All curve identifiers are from the
+     * <a href="https://www.iana.org/assignments/cose/cose.xhtml">
+     * IANA COSE registry</a>.
+     */
```

Comment:
Please add `TODO` here which says to port to using `KeystoreEngine.EcCurve` once the keystore-abstraction branch lands.